Assets/HotmaxEditorVR/builder/prop/scaleControl.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class scaleControl : MonoBehaviour
+{
+    private Vector3 growRate = new Vector3(.01f, .01f, .01f);
+
+    private float initialYPos;
+    private float currentYPos;
+
+    private Vector3 rayOrigin;
+    private Vector3 rayDirection;
+
+    private void OnEnable()
+    {
+        inputManager.trackedController2.TriggerClicked += triggerClicked;
+        init.scaleController.SetActive(true);
+    }
+
+    private void OnDisable()
+    {
+        inputManager.trackedController2.TriggerClicked -= triggerClicked;
+        init.scaleController.SetActive(false);
+    }
+
+    void triggerClicked(object sender, ClickedEventArgs e)
+    {
+        getInitialHitYPosition();
+    }
+
+    void Update()
+    {
+        setScaleControllerToFacePlayer();
+
+        if (inputManager.trackedController2.triggerPressed)
+        {
+            scaleObject();
+        }
+    }
+
+    void setScaleControllerToFacePlayer()
+    {
+        Transform scaleControllerHolder = init.scaleController.transform.parent.transform;
+
+        scaleControllerHolder.position = init.rotationGizmos.transform.position;
+        scaleControllerHolder.LookAt(init.vrCamera.transform);
+        scaleControllerHolder.eulerAngles = new Vector3(0, scaleControllerHolder.eulerAngles.y, 0);
+    }
+
+    void getInitialHitYPosition()
+    {
+        RaycastHit hit;
+        rayOrigin = inputManager.hand2.gameObject.transform.position;
```

Comment:
i think you're setting inputManager.hand2 as just _hand2 in another class.. i would keep that convention